osu.Game/Overlays/SettingsPanel.cs

```diff
@@ -209,6 +274,8 @@ public class SettingsSectionsContainer : SectionsContainer<SettingsSection>
         {
             public SearchContainer<SettingsSection> SearchContainer;
 
+            public new ScheduledDelegate Schedule(Action action) => Scheduler.AddDelayed(action, TransformDelay);
```

Comment:
Can (and probably should) be removed as unused.

Reply:
Makes me think about having an inspection for unused methods, usually any PR that adds a method would have it used elsewhere or at least tested, hmm...